i18n/locales/source/blockly-mooc/turtle.json

```diff
@@ -88,9 +91,8 @@
   "sizeParameter": "size",
   "step": "step",
   "tooFewColours": "You need to use at least %1 different colors for this puzzle.  You used only %2.",
-  "turnLeft": "turn left by",
-  "turnRight": "turn right by",
```

Comment:
Oh, I think we'll need to redo the sync in; the PRs that removed these strings had to be reverted

Reply:
Yeah I just realized that. Should we just wait on merging the down sync?

Reply:
we should probably cancel the down; since we (presumably) synced these removals up, the down step will also be removing them; but because the PRs were reverted, we actually still want to _use_ these strings, and so would prefer they remain translated

Reply:
I'm just not sure what another sync up will gain us if these strings are likely be removed once the reverted PRs are fixed. They won't be automatically translated again will they?

Reply:
Oh, right you are; I was thinking these strings were copied in playlab i18n, but I'm thinking of a different set of strings. Nevermind, then; we can continue with the sync down & out as normal. @nkiruka when do you expect to be able to un-revert those PRs? If it's within the next day or two, we can probably not bother to do a second sync up.

Reply:
By the end of the day, but let me know if changes are needed earlier so it isn't blocking the merge.